src/events/aws_sdk.js

```diff
@@ -110,6 +112,13 @@ const kinesisEventCreator = {
                 sequence_number: `${response.data.SequenceNumber}`,
             });
             break;
+        case 'putRecords':
+            eventInterface.addToMetadata(event, {
+                failed_record_count: `${response.data.FailedRecordCount}`,
+                shard_id: `${response.data.Records[0].ShardId}`,
```

Comment:
you use Records[0], it's always a first index,
Do we know 100% that important data stored in the first index?